app/views/videos/show.html.haml

```diff
@@ -7,6 +7,37 @@
         %header
           %h3= @video.title
           %h4=link_to @video.category.name, category_path(@video.category)
-          %span Rating: 4.5/5.0
+          %span
+            - if @video.average_rating != nil
```

Comment:
`!= nil` can be omitted. You are basically saying when its not not existing -, better say when its existing. `if @video.average_rating`